### PR TITLE
Clean up exposecontroller configuration

### PR DESCRIPTION
**Pull request by andrewazores**
Configure exposecontroller to watch services only in current namespace
as its own serviceaccount, removing setup-time configuration for which
namespace will be watched. This is preparation for bundling the operator
for operator catalogs, so that yaml definitions do not contain any
"REPLACE_IMAGE" or "REPLACE_PROJECT" type variables to be run through
sed.

### CODE REVIEW

File: deploy/operator.yaml [hunk 1]
```diff
@@ -16,8 +16,7 @@ spec:
       serviceAccountName: container-jfr-operator
       containers:
         - name: container-jfr-operator
-          # Replace this with the built image name
-          image: REPLACE_IMAGE
+          image: quay.io/rh-jmc-team/container-jfr:0.4.7
```
Comment (ebaron) [inline deploy/operator.yaml]: Should this be container-jfr-operator?

File: pkg/controller/containerjfr/resource_definitions/resource_definitions.go [hunk 2]
```diff
@@ -63,7 +63,7 @@ func NewPodForCR(cr *rhjmcv1alpha1.ContainerJFR) *corev1.Pod {
 func NewCoreContainer(cr *rhjmcv1alpha1.ContainerJFR) corev1.Container {
 	return corev1.Container{
 		Name:  cr.Name,
-		Image: "quay.io/rh-jmc-team/container-jfr:0.4.6-debug",
+		Image: "quay.io/andrewazores/container-jfr:latest",
```
Comment (ebaron) [inline pkg/controller/containerjfr/resource_definitions/resource_definitions.go]: Was this change intentional, or just for local testing?

Comment (andrewazores) [reply pkg/controller/containerjfr/resource_definitions/resource_definitions.go]: Local testing, fixed.

File: deploy/operator.yaml (hunk 1 above)
Comment (andrewazores) [reply deploy/operator.yaml]: Whoops, yes. Thanks for the catch!
